core/src/main/java/io/questdb/griffin/engine/QueryProgress.java

```diff
@@ -136,10 +141,15 @@ public static void logEnd(
         // as well as already converted to an immutable String, as needed to queue it up for handling
         // at a later time. For this reason, do not assign queryTrace.queryText = sqlText here.
         if (queryTrace != null && engine.getConfiguration().isQueryTracingEnabled()) {
+            if (queryTrace.shouldSubstituteBindVariables(engine.getConfiguration())) {
```

Comment:
why do we put this behind an extra config option?
to be able to switch it off if there is a performance impact?